sci-libs/pagmo/ChangeLog

```diff
@@ -1,7 +1,10 @@
 # ChangeLog for sci-libs/pagmo
-# Copyright 1999-2014 Gentoo Foundation; Distributed under the GPL v2
+# Copyright 1999-2015 Gentoo Foundation; Distributed under the GPL v2
 # $Header: $
 
+  22 Jul 2015;  <conrado@gentoo.org> pagmo-9999.ebuild:
```

Comment:
One minor issue. I think you forgot to set `ECHANGELOG_USER`. See [here](https://github.com/gentoo-science/sci/blob/master/CONTRIBUTING.md#recommended). Anyway, It's fine to correct the ChangeLog entry manually and to push without re-commit if it's only a ChangeLog fix.